services/src/kilda-pce/src/main/java/org/openkilda/pce/impl/algo/SimpleGetShortestPath.java

```diff
@@ -73,10 +77,10 @@ public SimpleGetShortestPath(AvailableNetwork network, SwitchId srcDpid, SwitchI
         this.end = network.getSwitches().get(dstDpid);
         this.allowedDepth = allowedDepth;
         if (this.start == null) {
-            logger.warn("SOURCE node doesn't exist. It isn't in the AVAILABLE network: {}", srcDpid);
+            log.warn("SOURCE node doesn't exist. It isn't in the AVAILABLE network: {}", srcDpid);
```

Comment:
It should lead to exception, not (only) to log message.

Reply:
Fixed.